authrization-service/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "authrization-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.799.0",
```

Comment:
`"aws-sdk"` should be in `devDependencies `. It will reduce bundle size. Lambda has this module under the hood.